src/applications/static-pages/translation.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
```

Comment:
Do you still need to disable this rule? I don't see any `console` statements in the file.